server/game/game.js

```diff
@@ -57,6 +58,10 @@ class Game extends EventEmitter {
         return player.constructor === Spectator;
     }
 
+    hasActivePlayer(playerName) {
+        return !!this.players[playerName] && !this.players[playerName].left;
```

Comment:
In general, I try to explicitly return a bool for methods that have `has` or `is` in the name.